orchagent/mirrororch.cpp

```diff
@@ -327,8 +369,10 @@ bool MirrorOrch::activateSession(const string& name, MirrorEntry& session)
     attr.value.u8 = MIRROR_SESSION_DEFAULT_IP_HDR_VER;
     attrs.push_back(attr);
 
+    // TOS value format is the following:
+    // DSCP 6 bits | ECN 2 bits
     attr.id =SAI_MIRROR_SESSION_ATTR_TOS;
```

Comment:
have one space between the '=' and the attribute name.
also fix the other places.